### PR TITLE
Ensure bungee is discussed and correct backend stuff.

### PR DESCRIPTION
**Pull request by Kichura**
1. This ensures we are targetting the 1.19+ backend servers as anything older than that will not ship with the enforce-secure-profile context in any way,

2. The Bungeecord / Waterfall proxy context should be included which tons of servers still use to this day and is best to be discussed about aswell if the user does not have velocity as their proxy,

3. Finally; FreedomChat has been chosen to be a comfortable plugin for usage in spigot and it's forks as it is genuinely safer for usage and doesn't cause breakage with other chat-related plugins.

*(Edits are welcomed)*

### CODE REVIEW

File: en-us/server-setup.md [hunk 1]
```diff
@@ -27,15 +27,14 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
...
+- **All 1.19+ backend servers**: set `enforce-secure-profile` to `false` in _server.properties_
   - This doesn't disable chat reporting by itself, but allows users to join without requiring them to sign their messages, to protect their privacy.
   - If no other measures are taken alongside this, _vanilla clients_ will see a warning toast in the top right corner and they will still sign the messages.
     - That means anyone can report them, but they cannot report FO users. To avoid those problems, use one of the plugins/mods below.
 - **Velocity**: set `force-key-authentication` to `false` in _velocity.toml_
   - Same comments apply as for "all servers" above.
-- **Paper/Spigot/Purpur**: install _one_ of the following plugins:
-  - [FreedomChat](https://modrinth.com/mod/freedomchat) - compatible with most chat plugins/vanilla chat, no config required
-  - [No Chat Reports](https://www.spigotmc.org/resources/no-chat-reports-spigot-1-19.102931/) - claims to be compatible with vanilla chat; not by the dev of the Fabric mod
+- **BungeeCord** / **Waterfall**: set `enforce_secure_profile` to `false` in _config.yml_
```
Comment (Madis0) [inline en-us/server-setup.md:36]: fix the spacing around "/" everywhere or nowhere

File: en-us/server-setup.md [hunk 2]
```diff
@@ -27,15 +27,14 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
...
+- **Paper/Spigot/Purpur**: install the [FreedomChat](https://modrinth.com/mod/freedomchat) plugin - no config required
```
Comment (Madis0) [inline en-us/server-setup.md:37]: let them have options

File: en-us/server-setup.md [hunk 3]
```diff
@@ -27,15 +27,14 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
 
 #### How to protect your users
 
-- **All servers**: set `enforce-secure-profile` to `false` in _server.properties_
+- **All 1.19+ backend servers**: set `enforce-secure-profile` to `false` in _server.properties_
```
Comment (Madis0) [inline en-us/server-setup.md:30]: could mention 1.18- too then that it doesn't need any changes

Comment (Madis0) [inline en-us/server-setup.md:30]:
```suggestion
- **All 1.19+ servers**: set `enforce-secure-profile` to `false` in _server.properties_
```

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: en-us/server-setup.md (hunk 2 above)
Comment (Kichura) [reply en-us/server-setup.md:38]: The second one is broken as it caused issues with chat-related plugins where things like HEX colors from 1.16+ becomes scrambled with 16 colors instead.

File: en-us/server-setup.md (hunk 1 above)
Comment (Kichura) [reply en-us/server-setup.md]: Resolved.

File: en-us/server-setup.md (hunk 3 above)
Comment (Kichura) [reply en-us/server-setup.md:30]: Resolved.

Comment (Madis0) [reply en-us/server-setup.md:30]: don't see the mention

**Pull request merged**